pom.xml

```diff
@@ -151,6 +151,11 @@
 				<type>pom</type>
 				<scope>import</scope>
 			</dependency>
+			<dependency>
+				<groupId>com.zaxxer</groupId>
+				<artifactId>HikariCP</artifactId>
+				<version>3.1.0</version>
```

Comment:
Can we parameterize this `version` value so that we can just update in one place if needed?